timescaledb/tutorials/financial-candlestick-tick-data/index.md

```diff
@@ -11,6 +11,13 @@ TimescaleDB using the OHLCV format. It also shows you how to download sample
 data containing real-world crypto tick transactions for cryptocurrencies like
 BTC, ETH, and other popular assets.
 
+<highlight type="note">
+Before you begin, make sure to have a TimescaleDB instance running locally or
+in cloud (see [installation options](/install/latest/)) and
+[Psql](/how-to-guides/connecting/psql) or any other PostgreSQL
```

Comment:
in a "note" blocks links don't seem to work properly (I need to specify the URL, can't use the shortcuts) @Loquacity

Reply:
that is correct. Still an issue because the highlight component is rendered separately from the main page and so the resulting HTML doesn't have access to the other logic. You need to include the full URL within highlight blocks.